src/server/__tests__/utils.test.tsx

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { handleReactRouterRedirection } from '../utils';
+
+describe('handleReactRouterRedirection', () => {
+  it('redirects to /fi/home 308', () => {
```

Comment:
very small thing but description could be `"redirects '/fi' to '/fi/home' (308 Permanent Redirect)"` and 
`"redirects '/' to '/fi/home' (302 Temporary Redirect)"`

Reply:
fixed :)